src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from 'react';
+import Area from './movie-area.js';
+import Header from './header.js';
+import movieData from './mock-data.js';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: []
+    };
+  }
+
+  componentDidMount() {
+    this.setState({movies: movieData.movies})
+  }
+
+  render() {
+    return (
+      <main className = 'App' >
+        <Header/>
+        <Area className='area' movies={this.state.movies} />
+      </main>
+    )
+  }
```

Comment:
good job keeping it simple, should be easy enough to refactor later when we fetch our data from an API.